sync-pipelines-rust/config-example.toml

```
@@ -0,0 +1,13 @@
+process_pool = 20
+transaction_id_prefix = "toad_"
+native_currency = "sepoliaeth"
+chain = "sepolia"
+erc20_contracts = [{usdc = "0x0"}]
+
+[last_sync_time]
```

```suggestion
[last_sync_time.files]
```
@horsey-defy what do you think?